Tests/IntegrationTests/Potsgrest/PostgrestResourceEmbeddingTests.swift

```diff
@@ -70,7 +70,7 @@ final class PostgrestResourceEmbeddingTests: XCTestCase {
       .eq("messages.channel_id", value: 1)
       .execute().value as AnyJSON
 
-    assertInlineSnapshot(of: res, as: .json) {
+    assertInlineSnapshot(of: { res }, as: .json) {
```

Comment:
🤔 why this change? didn't get it.

Reply:
Xcode shows error, but this not error...
I thought this is closure.

```swift
public func assertInlineSnapshot<Value>(
  of value: @autoclosure () throws -> Value?,
```